<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>串口调试助手</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_dataRecv">
     <property name="title">
      <string>数据接收区</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTextBrowser" name="textBrowser_dataRecv"/>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_showHex">
          <property name="text">
           <string>十六进制显示</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_showAppend">
          <property name="text">
           <string>追加显示</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_clearRecv">
          <property name="text">
           <string>清空</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
       <item>
        <widget class="QGroupBox" name="groupBox_param">
         <property name="title">
          <string>参数设置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_port">
                <property name="text">
                 <string>端口号</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_port">
                <item>
                 <property name="text">
                  <string>COM1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>COM2</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>COM3</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>COM4</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>COM5</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_stopBits">
                <property name="text">
                 <string>停止位</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_stopBits">
                <item>
                 <property name="text">
                  <string>1bit</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>1.5bit</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>2bit</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_dataBits">
                <property name="text">
                 <string>数据位</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_dataBits">
                <property name="currentIndex">
                 <number>3</number>
                </property>
                <item>
                 <property name="text">
                  <string>5bit</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>6bit</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>7bit</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>8bit</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item row="4" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_parity">
                <property name="text">
                 <string>校验</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_parity">
                <item>
                 <property name="text">
                  <string>无校验</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>奇校验</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>偶校验</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_baudRate">
                <property name="text">
                 <string>波特率</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_baudRate">
                <property name="currentIndex">
                 <number>7</number>
                </property>
                <item>
                 <property name="text">
                  <string>1200</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>2400</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>4800</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>9600</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>19200</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>38400</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>57600</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>115200</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="pushButton_refresh">
           <property name="text">
            <string>刷新</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_connect">
           <property name="text">
            <string>连接</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_dataSend">
       <property name="title">
        <string>数据发送区</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="plainTextEdit_dataSend"/>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_clearSend">
            <property name="text">
             <string>发送后清空</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_dataSend">
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
